bld/CLMBuildNamelist.pm

```diff
@@ -20,6 +20,7 @@
 # 2012-07-01  Kluzek           Add some common CESM namelist options
 # 2013-12     Andre            Refactor everything into subroutines
 # 2013-12     Muszala          Add Ecosystem Demography functionality
+# 2016-09-20  Oleson           Add ndep2 and ndep3 streams capability
```

Comment:
I'm assuming these names are for the new manure streams, but I think @ekluzek asked for these ndep2 & 3 to be replace with more descriptive names. would it be helpful to use consistent terminology?

Reply:
I guess the comment can be changed. Is this modification list still updated?

Reply:
@juliusvira as you can see we don't keep that list updated very often. And only sometimes update it with something that is pretty major. Since, git keeps track of the history, I don't tend to use this revision history.

So I'd recommend just deleting this line.